azurerm/resource_arm_public_ip_test.go

```diff
@@ -318,6 +338,23 @@ resource "azurerm_public_ip" "test" {
 `, rInt, location, rInt)
 }
 
+func testAccAzureRMPublicIPStatic_standard(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+    name = "acctestRG-%d"
+    location = "%s"
+}
+
+resource "azurerm_public_ip" "test" {
+    name = "acctestpublicip-%d"
+    location = "${azurerm_resource_group.test.location}"
+    resource_group_name = "${azurerm_resource_group.test.name}"
+		public_ip_address_allocation = "static"
```

Comment:
mixed tabs and spaces

Reply:
@paultyng I'm noticing this across multiple files in the provider... are tabs or spaces preferred?